addons/hr_timesheet/static/src/scss/qr_modal.scss

```diff
@@ -0,0 +1,3 @@
+.modal-content .modal-footer {
```

Comment:
That way you hide all the boders of all the modal when your module is installed.

Your rule should be more specific ;)